RIE/RIE185bisand270bis.xml

```diff
@@ -159,35 +160,205 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                    <lb n="7"/> ἡμετέρους ἀδελφοὺς Σαζανᾶν <expan><abbr>κ</abbr><ex>αὶ</ex></expan> τὸν Ἀδιφᾶν
+                    <lb n="8"/> τούτους πολεμῆσαι καὶ παραδεδωκότων αὐ<lb n="9" break="no"/>
+                    τῶν ὑποτάξαντες αὐτοὺς ἤγαγον πρὸς ἡμᾶς
+                    <lb n="10"/> μετὰ καὶ τοῦ ὄχλου αὐτῶν ὅλου καὶ τῶν θρεμμά<lb n="11" break="no"/>
+                    των, βοῶν γριβ καὶ προβάτων ςσκδ καὶ νω<lb n="12" break="no"/>
+                    τοφόρων χοζˊ, θρέψαντες αὐτοὺς βόεσίν τε
+                    <lb n="13"/> καὶ ἐπισιτισμῷ <expan><abbr>ἀννών</abbr><ex>ης</ex></expan> <expan><abbr>οἶν</abbr><ex>ῳ</ex></expan>
+                    τε <expan><abbr>κ</abbr><ex>αὶ</ex></expan> ὑδρομέλιτι,
+                    <lb n="14"/> ζύτῳ <expan><abbr>κ</abbr><ex>αὶ</ex></expan> ὑδρεύμασιν είς χορτασίαν ἐπὶ μῆνας
+                    <lb n="15"/> τέσσαρας οἵτιναις ἦσαν τὸν ἀριθμόν δυ <expan><abbr>κ</abbr><ex>αὶ</ex></expan>
+                    <lb n="16"/> ἀννωνευόμενοι καθ'ἑκάστην ἡμέραν ἄρτους
+                    <lb n="17"/> σιτίνους <expan><abbr>μυ</abbr><ex>ριάδας</ex></expan> β β, ἄχρεις αὐτοὺς μετοικίσωμεν˙
+                    <lb n="18"/> τούτους οὖν ἐνεχθέντας πρὸς ἡμᾶς, δωρησά<lb n="19" break="no"/>
+                    μενοι αὐτοῖς πάντα τὰ ἐπιτήδια καὶ ἀμφιάσαν<lb n="20" break="no"/>
+                    τες, μετοικίσαντες αὐτοὺς κατεστήσαμεν ἴς τι<lb n="21" break="no"/>
+                    να τόπον τῆς ἡμετέρας χώρας καλούμενον
+                    <lb n="22"/> Μάτλια καὶ ἐκελεύσαμεν αὐτοὺς πάλιν ἀννω<lb n="23" break="no"/>
+                    νεύεσθαι ἐπὶ τοῖς ἐκῐσε τόποις, παρασχόμενοι
+                    <lb n="24"/> ἑκάστῳ βασιλίσκῳ αὐτῶν βόας δρϟ ἤ
+                    <lb n="25"/> ὡς γύνεσθαι τοῖς ἕξασιν βασιλίσκοις βόας
+                    <expan><abbr>μυ</abbr><ex>ριάδας</ex></expan> β́ ερμ́ ̇
+                    <lb n="26"/> υπὲρ δὲ εὐχαριστίας τοῦ μαι γεννήσαντος
+                    <lb n="27"/> ἀνικήτου Ἄρεως ἀνεθήκαμεν αὐτῳ ἀνδρι<lb n="28" break="no"/>
+                    άντα χρὺσαιον ἕνα <expan><abbr>κ</abbr><ex>αὶ</ex></expan> ἀργύραιον ἔνα <expan><abbr>κ</abbr><ex>αὶ</ex></expan> χαλκαίους
+                    <lb n="29"/> τρὶς καὶ ἀνέθηκα ταύτην τὴν στήλην καὶ παρε<lb n="30" break="no"/>
+                    θέμην αὐτὴν τῷ οὐρανῷ καὶ τῇ γῇ καὶ τῷ μαι
+                    <lb n="31"/> γεννήσαντι ἀνικήτῳ Ἀρέει˙ ἔι τις οὖν τοῦτον
+                    <lb n="32"/> ἀδικῆσαι βουληθῇ, ἐξολεθρεύσῃ αὐτὸν ὁ θὲ<supplied reason="lost">ὸς</supplied>
+                    <lb n="33"/> τοῦ αὐρανοῦ καὶ τῆς γῆς ἐκρίζων καὶ μὴ ὑπάρ<lb n="34" break="no"/>
+                    <supplied reason="lost">χ</supplied>ῃ αὐτοῦ ὄνομα ἐν γῇ ζώντων˙ καὶ <expan><abbr>εὐχαρι</abbr><ex>σ</ex></expan><lb n="35" break="no"/>
+                    τήσαντες ἀνεθήκαμεν ἐπ' ἀγαθῷ˙ <g ref="palm"/>
+                    <lb n="36"/> εὐθὺς ἀνεθήκαμεν δὲ τῷ ἀνικήτῳ
+                    <lb n="37"/> Ἀρέει CΟΥʿΑΤΕ καὶ ΒΕΔΙΕ.
```

Comment:
are these upper case because unclear? if so, they may be marked accordingly

Reply:
The commentaire says that the inscription ends with an ethiopic consecration transliterated into Greek. Maybe because there were not equivalent formulas. So, I just copy the text in the volume.

Reply:
please have a go at marking that up appropriately as the book say. I would, for example, mark up at least the change of language with `<foreign>`